src/main/java/org/agbrothers/englishbot/processing/processor/lesson/english/EnglishAudioLessonProcessor.java

```diff
@@ -0,0 +1,70 @@
+package org.agbrothers.englishbot.processing.processor.lesson.english;
+
+import org.agbrothers.englishbot.constant.State;
+import org.agbrothers.englishbot.entity.Lesson;
+import org.agbrothers.englishbot.entity.ResponseMessage;
+import org.agbrothers.englishbot.entity.Word;
+import org.agbrothers.englishbot.processing.ProcessingExchange;
+import org.agbrothers.englishbot.service.texttospeech.TextToSpeechClient;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+import org.telegram.telegrambots.meta.api.objects.InputFile;
+
+import static org.agbrothers.englishbot.constant.CommonPhrase.MAKE_CHOICE_AUDIO;
+
+@Component
+public class EnglishAudioLessonProcessor extends AbstractEnglishLessonProcessor {
+
+    @Autowired
+    private TextToSpeechClient voicerssTextToSpeechClient;
+
+    @Override
+    public String getValueToTranslate(Word word) {
+        return "";
```

Comment:
why not null?

Reply:
return "" because of logic in appendResponseMessageText.  I left a TODO comment here to fix this within https://agbrothers.atlassian.net/browse/EN-67